rest-server/src/models/job.js

```diff
@@ -397,27 +397,28 @@ class Job {
   }
 
   generateYarnContainerScript(data, idx) {
+    let tasksNumber = 0;
+    for (let i = 0; i < data.taskRoles.length; i ++) {
+      tasksNumber += data.taskRoles[i].taskNumber;
+    }
     const yarnContainerScript = mustache.render(
         yarnContainerScriptTemplate, {
           'idx': idx,
```

Comment:
>idx [](start = 11, length = 3)

"taskIndex"?

Reply:
it's index in the array, just to differ task roles.

Reply:
Suggest not use abbreviation as other key names here are not.

---
In reply to: [184361207](https://github.com/Microsoft/pai/pull/467#discussion_r184361207) [](ancestors = 184361207)